testsuite/features/core_srv_create_repository.feature

```diff
@@ -124,3 +162,11 @@ Feature: Add a repository to a channel
     And I follow "Test-Channel-x86_64"
     And I follow "Packages" in the content area
     Then I should see a "blackhole-dummy" text
+
+  Scenario: Reposync handles wrong encoding on DEB attributes
```

Comment:
DEB -> Debian
conditional ?

Reply:
In this case it "DEB" refers to the pkg format, same as "RPM" in the previous scenario.